### ISSUE TITLE
[Adaptive Cards> Input.json]: Validation is not set for the fields (Email and Number) present under Input Cards. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#205]
### Target Platforms

Android

### SDK Version

1.0.1 (2)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 1.0.1 (2)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Input.JSON'.
8. Navigate the card using swipe gesture.
9. Navigate to the Email or Phone Number field using swipe gesture.
10. Enter some incorrect value and perform right swipe and navigate to submit button.
11. Double tap to activate it and observe the issue.

**Actual Result:**
No validation is set for the Email and Phone Number fields present under Input card as after entering the incorrect value on activating the submit button no error message is getting displyed.

**Expected Result:**
Validation should be set for the fields (Email and Phone number) so that after entering the incorrect value, error message should be displayed.

**User Impact:**
It will impact the user as they will block to input the valid values due to this there might be chance to enter the incorrect values.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/d0184a2f-1788-466f-aa7a-53afebfd8d57

### Card JSON

```JSON
NA
```
